secrets.sample.js

```diff
@@ -0,0 +1,2 @@
+// You can register your app and get a client ID at https://apps.dev.microsoft.com
+window.ClientId = "<insert id here>";
```

Comment:
Graph explorer can still run locally if they don't want to setup auth. The login page will show a client id not defined error, which should point devs in the right direction.